### ISSUE TITLE
Support for VS2022

### ISSUE THREAD

**Issue by RainOrigami** [munyabe/FormatAllFiles#16]
Hello

Will there be support for VS2022?
I am not sure why formatting is not already part of the code cleanup process that is included in VS and as long as this is missing the extension will be very valuable.

Thank you.

**Comment by InvalidCast44** [comment]
i reealy do agree

**Comment by Havunen** [comment]
ping @munyabe

**Comment by Havunen** [comment]
I have forked this plugin for Visual Studio 2022 https://github.com/Havunen/FormatAllFiles its not yet available in market place, but you can build it in release mode and it will generate VSIX installer file for the plugin. Only Visual Studio 2022 is supported

**Comment by nachum-shmilovitz** [comment]
I compile it and install it on VS 2022 
work like a charm
Thx

**Comment by tacosontitan** [comment]
> I have forked this plugin for Visual Studio 2022 https://github.com/Havunen/FormatAllFiles its not yet available in market place, but you can build it in release mode and it will generate VSIX installer file for the plugin. Only Visual Studio 2022 is supported

Create a PR?

**Comment by nachum-shmilovitz** [comment]
Hi
Thx
build it and it is working on 2022
Thx


Nachum

From: Taco (タコス) ***@***.***>
Date: Thursday, 22 September 2022 at 1:36
To: munyabe/FormatAllFiles ***@***.***>
Cc: Nachum Shmilovitz ***@***.***>, Comment ***@***.***>
Subject: Re: [munyabe/FormatAllFiles] Support for VS2022 (Issue #16)

I have forked this plugin for Visual Studio 2022 https://github.com/Havunen/FormatAllFiles<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHavunen%2FFormatAllFiles&data=05%7C01%7Cnshmilovitz%40attentigroup.com%7Cb99cbde145f34bb0d25908da9c21bd5f%7Ccb91552f7d2a449d95b84638c423f880%7C0%7C0%7C637993965997417811%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=pl4xqCXZA0CvFdNDFThFoWwvqrajc%2BUvznxpuOb7i2Y%3D&reserved=0> its not yet available in market place, but you can build it in release mode and it will generate VSIX installer file for the plugin. Only Visual Studio 2022 is supported

Create a PR?

―

**Comment by tacosontitan** [comment]
> Hi Thx build it and it is working on 2022 Thx Nachum

I understand that much, but creating a PR would allow the project owner to update the one in the marketplace too. As a result more people will have access to this plugin.

**Comment by nachum-shmilovitz** [comment]
Hi,

never did it here

So, if I understand u right, you want me to create PR that say that this plugin does not support VS2022, so you can solve it and merge the PR to the master branch?

Nachum

From: Taco (タコス) ***@***.***>
Date: Thursday, 22 September 2022 at 14:12
To: munyabe/FormatAllFiles ***@***.***>
Cc: Nachum Shmilovitz ***@***.***>, Comment ***@***.***>
Subject: Re: [munyabe/FormatAllFiles] Support for VS2022 (Issue #16)

Hi Thx build it and it is working on 2022 Thx Nachum

I understand that much, but creating a PR would allow the project owner to update the one in the marketplace too. As a result more people will have access to this plugin.

―

**Comment by tacosontitan** [comment]
> Hi, never did it here So, if I understand u right, you want me to create PR that say that this plugin does not support VS2022, so you can solve it and merge the PR to the master branch? Nachum

No. @Havunen created a fork of the repository and got it working with VS2022. I was asking if they would create a PR so that this repo can be updated with the fix.

**Comment by Havunen** [comment]
It might be that this repository is actually abandoned as there has not been activity from the maintainer for 4 years. Maybe I could publish a fork of this plugin to MarketPlace just for VS2022

**Comment by Havunen** [comment]
Its now available in market place with name "FormatAllFilesPlus"

https://marketplace.visualstudio.com/items?itemName=Havunen.FormatAllFilesPlus-1
